### DISCUSSION TITLE
Is it possible to set up anonymized dns with DNScrypt in Adguard Home?

### DISCUSSION THREAD

**Discussion by Z0pyrus** [AdguardTeam/AdGuardHome#2971] [Q&A] [answered]
Is it possible to set up anonymized dns with DNScrypt in Adguard Home?

**Comment by ameshkov** [comment, marked as answer]
Nope. AdGuard supports DNSCrypt protocol, but not Anonymized DNSCrypt.

There's a feature request about it: https://github.com/AdguardTeam/AdGuardHome/issues/1226
